src/OracleService/OracleService/config.json

```diff
@@ -9,7 +9,7 @@
       "Timeout": 5000
     },
     "NeoFS": {
-      "EndPoint": "127.0.0.1:8080",
+      "EndPoint": "http://127.0.0.1:8080",
```

Comment:
Force to be `https`?

Reply:
We can use `http` for now. Because neofs node use `http` in general like our `RpcServer`.